test/decompiler/reference/jak1/engine/entity/entity_REF.gc

```diff
@@ -1881,7 +1879,7 @@
       (reset-actors 'debug)
       0
       (disable-irq)
-      (.mtc0 Count r0)
+      (.mtc0 Count 0)
```

Comment:
here's a good example where we'd actually want to see the `.mtc0` instruction to understand what's happening. They measure elapsed time using the `Count` register, then use this to decide how many actors to spawn.